src/system/AdminModule/Controller/AdminInterfaceController.php

```diff
@@ -0,0 +1,469 @@
+        $caller['category'] = \ModUtil::apiFunc('ZikulaAdminModule', 'admin', 'getCategory', array(
+            'cid' => $cid
+        ));
+        
+        return $this->render("ZikulaAdminModule:AdminInterface:breadcrumbs.html.twig", [
+            'caller' => $caller
+        ]);
+    }
+
+    /**
+     * @Route("/developernotices")
+     *
+     * Add developer notices
+     *
+     * @return Response symfony response object
+     */
+    public function developernoticesAction()
+    {
+        if (! $this->hasPermission('ZikulaAdminModule::', '::', ACCESS_ADMIN)) {
+            throw new AccessDeniedException();
+        }
+        
+        $modvars = $this->get('zikula_extensions_module.api.variable')->getAll('ZikulaThemeModule');
+        $data = [];
+        $data['mode'] = $this->get('kernel')->getEnvironment() === 'dev';
+        if ($data['mode'] == 'dev') {
+            $data['legacy'] = array(
+                'status' => true,
+                'cssjscombine' => $modvars['cssjscombine'],
+                'render' => array(
+                    'compile_check' => array(
+                        'state' => $modvars['render_compile_check'],
+                        'title' => $this->__('Compile check')
+                    ),
+                    'force_compile' => array(
+                        'state' => $modvars['render_force_compile'],
+                        'title' => $this->__('Force compile')
+                    ),
+                    'cache' => array(
+                        'state' => $modvars['render_cache'],
+                        'title' => $this->__('Caching')
+                    )
+                ),
+                'theme' => array(
+                    'compile_check' => array(
+                        'state' => $modvars['compile_check'],
+                        'title' => $this->__('Compile check')
+                    ),
+                    'force_compile' => array(
+                        'state' => $modvars['force_compile'],
+                        'title' => $this->__('Force compile')
+                    ),
+                    'cache' => array(
+                        'state' => $modvars['enablecache'],
+                        'title' => $this->__('Caching')
+                    )
+                )
+            );
+        }
+        return $this->render("ZikulaAdminModule:AdminInterface:developernotices.html.twig", [
+            'developer' => $data
+        ]);
+    }
+
+    /**
+     * @Route("/securityanalyzer")
+     *
+     * Add security analyzer
+     *
+     * @return Response symfony response object
+     */
+    public function securityanalyzerAction()
+    {
+        $data = [];
+        $data['legacy'] = array(
+            // check for magic_quotes - this was removed php 5.4.0
+            'magic_quotes_gpc' => get_magic_quotes_gpc(),
+            // check for register_globals - this was removed php 5.4.0
+            'register_globals' => ini_get('register_globals'),
```

Comment:
If it has been removed with PHP 5.4 - and the core requires PHP 5.4 - can these checks be removed alltogether?